content/en/serverless/google_cloud_run/_index.md

```diff
@@ -107,96 +140,154 @@ CMD ["rails", "server", "-b", "0.0.0.0"] (adapt this line to your needs)
+  --allow-unauthenticated
+
+```
 
-   Under **Reference method**, select **Exposed as environment variable**.
+{{% /tab %}}
+{{< /tabs >}}
 
-   Under the **Environment variables** section, ensure that the name is set to `DD_API_KEY`.
+These are instructions for deploying a Cloud Run service using standard GCP tools. If you have other systems in place
+for managing container images, secrets, or deployments, you may use those instead.
 
-### Custom metrics
-You can submit custom metrics using a [DogStatsd client][7].
+### Results
 
-**Note**: Only `DISTRIBUTION` metrics should be used.
+You should be able to see metrics and traces of your Cloud Run application in the Datadog UI!
+You can submit custom metrics using a [DogStatsd client][7]. Only `DISTRIBUTION` metrics should be used.
+
+### Logs
+
+If you use [GCP integration][1] your logs are already being collected. Alternatively, you can set the `DD_LOGS_ENABLED`
+environment variable to true to capture application logs through the serverless instrumentation.
 
 ### Advanced options and configurations
 
 #### Environment variables
 
-| Variable | Description |
-| -------- | ----------- |
-| `DD_SITE` | [Datadog site][8]. |
+| Variable          | Description                                                             |
+|-------------------|-------------------------------------------------------------------------|
+| `DD_SITE`         | [Datadog site][8].                                                      |
 | `DD_LOGS_ENABLED` | When true, send logs (stdout and stderr) to Datadog. Defaults to false. |
-| `DD_SERVICE` | See [Unified Service Tagging][9]. |
-| `DD_VERSION` | See [Unified Service Tagging][9]. |
-| `DD_ENV` | See [Unified Service Tagging][9]. |
-| `DD_SOURCE` | See [Unified Service Tagging][9]. |
-| `DD_TAGS` | See [Unified Service Tagging][9]. |
+| `DD_SERVICE`      | See [Unified Service Tagging][9].                                       |
+| `DD_VERSION`      | See [Unified Service Tagging][9].                                       |
+| `DD_ENV`          | See [Unified Service Tagging][9].                                       |
+| `DD_SOURCE`       | See [Unified Service Tagging][9].                                       |
+| `DD_TAGS`         | See [Unified Service Tagging][9].                                       |
+
+## Troubleshooting
+
+#### SSL
 
-## Log collection
+This integration depends on your runtime having a full SSL implementation. If you are using a slim image for Node, you
+may need to add the following command to your Dockerfile to include certificates.
 
-You can use the [GCP integration][1] to collect logs. Alternatively, you can set the `DD_LOGS_ENABLED` environment variable to true to capture application logs through the Agent.
+```
+RUN apt-get update && apt-get install -y ca-certificates
+```
+
+#### Buildpack fail with 'Failed to run /bin/build: for Python, an entrypoint must be manually set, either with "GOOGLE_ENTRYPOINT"'
```

Comment:
either with "GOOGLE_ENTRYPOINT" or ...??? is this sentence finished? also can we use a keyword for title? this very long sentence looks weird when rendered as a bolded title.

Reply:
I will remove this part since it's not related to the datadog integration but general debugging